attributes/default.rb

```diff
@@ -82,7 +82,6 @@
 default['librenms']['rrdcached']['enabled'] = false
 default['librenms']['rrdcached']['config_file'] = '/etc/sysconfig/rrdcached'
 default['librenms']['rrdcached']['options'] = '-w 1800 -z 1800 -f 3600 -B -R -j /var/tmp -l unix:/var/run/rrdcached/rrdcached.sock -t 4 -F'
-default['librenms']['rrdcached']['path'] = "node['librenms']['path']/rrd"
```

Comment:
why you remove that line ?

Reply:
this attribute is not used, there is the equivalent called ['librenms']['rrd_dir']